index.html

```diff
@@ -41,7 +41,180 @@
             </div>
         </div>
         </section>
-        
     </div>
+    <section class="works">
+        <p id="Recent-Works-Text">My Recent<br>Works</p>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
+        </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="img-Plaholder"></div>
+        <div class="project-detail">
+        <p id="card-text">Multi-Post Stories Gain+Glory</p>
+        <div class="work-category">
+            <ul id="work-list">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul> 
+        </div>
+        <div id="see-project-btn">
+            <p id="see-project">See Project</p>
```

Comment:
Kindly use a `<button>` or `<li><a>` for this. This is for semantic purposes and so that screen readers can understand your page